### PR TITLE
Multiple updates - emulator, version detection, aggregator, beautification

### PR DESCRIPTION
**Pull request by m-1-k-3**
**Multiple updates:**
* The emulator is able to detect the root filesystem more accurate
* The emulator is able to handle multiple root paths
* The emulator is able to identify broken rootfs and do not handle them
* The detection of weak binary functions in ARM binaries had some issues -> fixed
* Removed the exit on log mounts -> needed for docker
* Fixed a bug in the binary protection calculation of the aggregator
* Included new versions and fixed some version detection strings
* Included interesting bins into aggregator
* Cleaned the output of cwe-checker module:
image
* The aggregator now also gives an overview of cwe-checker results
image
* The aggregator now also shows strcpy and system call overview:
image

### CODE REVIEW

File: modules/F50_base_aggregator.sh [hunk 1]
```diff
@@ -76,58 +80,100 @@ F50_base_aggregator() {
...
-    fi
-  fi
+    fi  
+  fi 
 
   print_output ""
-  if [[ -f "$LOG_DIR"/"$CVE_AGGREGATOR_LOG" ]]; then
+  if [[ "$S30_VUL_COUNTER" -gt 0 || "$CVE_COUNTER" -gt 0 || "$EXPLOIT_COUNTER" -gt 0 ]]; then
     print_output "[*] Identified the following version details, vulnerabilities and exploits:"
     print_output "$(grep "\[+\] Found version details" "$LOG_DIR"/"$CVE_AGGREGATOR_LOG" 2>/dev/null)"
```
Comment (p4cx) [inline modules/F50_base_aggregator.sh:178]: `print_output` can't modify the string correctly (colors): 

image

**Review by p4cx — CHANGES_REQUESTED** [review]
Some small fixes, then it will be merged. Thank you :)

File: modules/S10_binaries_check.sh [hunk 2]
```diff
@@ -143,36 +143,64 @@ objdump_disassembly()
...
+              #      COUNT_STRLEN=$(grep -c "[[:blank:]]bl[[:blank:]]strlen" "$FUNC_LOG"  2> /dev/null)
+              #      (( STRCPY_CNT="$STRCPY_CNT"+"$COUNT_FUNC" ))
+              #    elif [[ "$FUNCTION" == "mmap" ]] ; then
+              #      # Test source: https://www.golem.de/news/mmap-codeanalyse-mit-sechs-zeilen-bash-2006-148878-2.html
+              #      # Check this testcase. Not sure if it works in all cases! 
+              #      COUNT_MMAP_OK=$(grep -c "cm.*r.*,\ \#[01]" "$FUNC_LOG"  2> /dev/null)
+              #    fi
+              #    output_function_details
+              #  fi
+              #fi
```
[the remark this answers is not in the document]
Comment (m-1-k-3) [reply modules/S10_binaries_check.sh:176]: you are right ... I will remove this

File: modules/F50_base_aggregator.sh (hunk 1 above)
Comment (m-1-k-3) [reply modules/F50_base_aggregator.sh:178]: I will clean this a bit

**Review by p4cx — APPROVED** [review]
Thank you

**Pull request merged**